exercises/1-the-manual-menace/README.md

```diff
@@ -65,15 +65,7 @@ If you're feeling confident and don't want to follow the step-by-step guide thes
 ### Part 1 - Create your cloud workspace
 > _Create your cloud ide environment using Che_
 
-1. To create your cloud ide environment, open a web browser using the following URL:
-
-```
-https://codeready-workspaces.apps.<DOMAIN_FOR_YOUR_CLASS>/dashboard/#/load-factory?name=DO500%20Template&user=admin
-```
-
-<p class="tip">
-<b>NOTE</b> - Complete URL should be replaced with the one you've been provided by the instructor.
-</p>
+1. To create your cloud ide environment, open a web browser using the Code Ready Workspaces (CRW) URL provided by your instructor.
```

Comment:
@jtudelag I still like leaving the URL in here, but agree with clarifying that <DOMAIN_FOR_YOUR_CLASS> will be provided by the instructor. This way it's just a matter of copy and pasting the link from the docs and replacing <DOMAIN_FOR_YOUR_CLASS>. Could we leave this in and just clarify what needs replaced and what will be provided by the instructor?

Reply:
I do like keeping the link here to "indicate" which one they should be looking for from the instructor. Agreed that the wording could potentially be better, although the `NOTE` is somewhat already explaining what this is and how it should be treated.

Reply:
If the link is gonna be provided by the instructor anyway, why keeping it here?
Besides, the link placeholder is wrong. At least in the last to DO500 delivered in EMEA recently, they don't follow the pattern `codeready-workspaces.apps.<DOMAIN_FOR_YOUR_CLASS>`

It confused most of the attendees of the class. IMHO it just add noise and confuses people. Instructions are clearer without it.

Reply:
As @jtudelag mentioned the URL pattern is wrong as this is not following the instructions we use to deploy CRW:

https://github.com/rht-labs/enablement-codereadyworkspaces#openshift-container-platform-v3

We are always providing this info to the students, we use Slack to pin important info to the channel like OCP Console URL, GitLab URL, Users' login and CodeReady URLs, so I don't really see the point on having information there. During the last 3 DO500 this was confusing the audience and they gave us this feedback each time.